CHANGELOG.md

```diff
@@ -1,5 +1,49 @@
 # CHANGELOG
 
+## v0.5.0
+
+### What's New
+
+**Manage EdgeX Foundry system in OpenYurt in a cloud-native, non-intrusive way**
+
+- yurt-edgex-manager
+  Yurt-edgex-manager is OpenYurt to manager EdgeX lifecycle controller, it contains one CR (Custom Resource) to represents one EdgeX deployment.
```

Comment:
How about this:

Yurt-edgex-manager enable OpenYurt to be able to manage the EdgeX lifecycle. Each Edgex CR (Custom Resource) stands for an EdgeX instance. Users can deploy/update/delete EdgeX in OpenYurt cluster by operate the EdgeX CR directly.

Reply:
more readable, updated.